spec/lib/pre_assembly/media_spec.rb

```diff
@@ -24,9 +24,7 @@
         expect(noko_doc(dobj2.send(:create_content_metadata))).to be_equivalent_to noko_doc(exp_xml_object_bb222bb2222)
       end
     end
-  end
 
-  describe '#create_content_metadata' do
```

Comment:
per the rubocop complaint, i just collapsed the two describe blocks for this method into one.